<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatWindow</class>
 <widget class="QWidget" name="chatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat聊天室</string>
  </property>
  <widget class="QTextEdit" name="edit_text">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>511</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="send_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>470</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="message_text">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
